manifests/apache.pp

```diff
@@ -114,7 +123,7 @@
       cert_owner => 'root',
       cert_group => $group,
       cert_mode  => '0440',
-      require    => Cert[$apache_cert_name],
+      require    => File["${apache_cert_path}.crt"],
```

Comment:
should this also subscribe to the `.key`? Granted, I can't really think of a case where the key changes but not the cert, but better safe than sorry while debugging weird cert issues in the future?